### DISCUSSION TITLE
请教个问题，鼠标手势 + MPV + youtube-dl 怎么通用gotox 使用

### DISCUSSION THREAD

**Discussion by nbzdwss78** [SeaHOH/GotoX#37] [问答/咨询]

**Comment by SeaHOH** [comment]
MPV 和 youtube-dl 默认都会使用系统证书，导入系统就好。下面是代理设置。
```ini
#mpv.conf
#MPV 设置我一般停用 youtube-dl，鼠标调用时添加“--ytdl”参数
ytdl=no
ytdl-raw-options='"sub-lang"="zh-Hans,zh-Hant,ja,en","proxy"="HTTP://127.0.0.1:8087"'
```

**Comment by SeaHOH** [comment]
对了，MPV 还需要设置环境变量。你也可以把这个变量想办法在调用 MPV 时加入，或者用 bat 脚本。
`http_proxy: http://127.0.0.1:8087`

**Comment by SeaHOH** [comment]
都可以，不过插件可以直接获取链接地址，单独的手势程序还需要一个链接的传递过程，比如先复制链接。
我用是火狐 FireGestures，搜了一下 Chrome 还是有不少的类似插件的，你选一款支持调用其它程序的手势插件就好。
